spec/models/manageiq/providers/ansible_role_workflow_spec.rb

```diff
@@ -178,19 +178,28 @@
     let(:state) { "execute" }
     let(:response_async) { Ansible::Runner::ResponseAsync.new(:base_dir => "/path/to/results") }
 
-    it "ansible-runner succeeds" do
-      response_async = Ansible::Runner::ResponseAsync.new(:base_dir => "/path/to/results")
```

Comment:
For some reason these tests overrode the `let` above with the exact same value, so I cleaned that up also.